ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,50 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, Algorithm, noncommoninds
+using ITensorNetworks:
+  ITensorNetworks, ITensorNetwork, vertextype, vertex_data, contraction_sequence
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function to_einexpr(ts::Vector{ITensor})
+  IndexType = Any
+
+  tensor_exprs = EinExpr{IndexType}[]
+  inds_dims = Dict{IndexType,Int}()
+
+  for tensor_v in ts
+    inds_v = collect(inds(tensor_v))
+    push!(tensor_exprs, EinExpr{IndexType}(; head=inds_v))
+    merge!(inds_dims, Dict(inds_v .=> size(tensor_v)))
+  end
+
+  externalinds_tn = collect(noncommoninds(ts...))
```

Comment:
```suggestion
  externalinds_tn = reduce(noncommoninds, ts)
```
Splatting a big vector can cause trouble for the compiler.